### PR TITLE
dawg-python 0.7.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**
License: https://github.com/pytries/DAWG-Python/blob/0.7.2/LICENSE
Requirements: https://github.com/pytries/DAWG-Python/blob/0.7.2/setup.py

Notes:
- the package is a dependency of `pymorph3` and it's for resolving this issue https://github.com/conda-forge/spacy-models-feedstock/issues/2

### CODE REVIEW

**Review by ELundby45 — APPROVED** [review]

**Review by varlackc — COMMENTED** [review]
We might want to remove the `abs.yaml` file from the recipe before we merge the package to defaults.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,27 +11,35 @@ source:
...
+  requires:
+    - pip
+  commands:
+    - pip check
 
 about:
   home: https://github.com/kmike/DAWG-Python/
   license: MIT
   license_family: MIT
   license_file: LICENSE.txt
```
Comment (Jrice1317) [inline recipe/meta.yaml]:
```suggestion
  license_file: LICENSE
```

Comment (Jrice1317) [reply recipe/meta.yaml]: https://github.com/pytries/DAWG-Python/blob/0.7.2/LICENSE

**Review by Jrice1317 — COMMENTED** [review]
I agree with removing the `abs.yaml`. I also want to note the license file does not have the `.txt` extension so we may want to remove that. Another minute note is the `summary` and `description` are the same, but other than that, LGTM!

**Review by boldorider4 — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml]: Fixed

**Pull request merged**